main.go

```diff
@@ -123,10 +126,35 @@ func main() {
 			log.Fatalf("FATAL: unable to open %s %s", opts.HtpasswdFile, err)
 		}
 	}
+	ostream := os.Stdout
+	if opts.LogFilePath != "" {
+		ostream = openLog(opts.LogFilePath)
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, os.Interrupt, syscall.SIGHUP)
+
+		go func(path string) {
+			for {
+				<-c
+				fmt.Println("Recreating log stream...")
+				ostream.Close()
```

Comment:
@ChrisPRobinson Thanks for the quick review! Let me know if the latest commit it sufficient.